app/models/work.rb

```diff
@@ -0,0 +1,64 @@
+class Work < ApplicationRecord
+  has_many :votes, dependent: :destroy
+  has_many :users, through: :votes
+  
+  validates :title, presence: true
+  validates :category, presence: true
+  validates_uniqueness_of :title, scope: :category, message: "has already been taken"
+
+  def self.get_in_category_by_votes(category_name)
+    valid_categories = ["album", "book", "movie"]
+    raise ArgumentError.new("#{category_name} is not valid") if !valid_categories.include?(category_name)
+
+    num_category_works = self.where(category: category_name).count
+
+    return self.where(category: category_name).includes(:votes).max_by(num_category_works) {
+      |work| work.votes.count
+    }
```

Comment:
Just noting it's better to sort the works at the database-level with things like `.order` as Postgres is more efficient at filtering and sorting records than Ruby/Rails are.